<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardView</class>
 <widget class="QFrame" name="CardView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>163</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QLabel" name="numberTopLeft">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(1,1,1,0);</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="numberBottomRight">
   <property name="geometry">
    <rect>
     <x>131</x>
     <y>160</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255,0)</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>66</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/icons/resources/piko.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="iconTop">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>10</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #f00;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="iconBottom">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #f00;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: orange;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>imageLabel</zorder>
  <zorder>numberBottomRight</zorder>
  <zorder>numberTopLeft</zorder>
  <zorder>iconTop</zorder>
  <zorder>iconBottom</zorder>
  <zorder>back</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
